.eslintrc.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  root: true,
+  extends: '@react-native-community',
```

Comment:
Haha but why eslint? :D

Reply:
@perry-mitchell dont know really :D came with the RN upgrade. I'll check if it's not important I'll just remove it.

Reply:
Let's kill it.. :D